index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Technical Documentation</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+
+<nav id="navbar">
+  <header>What is HTML</header>
+  <a class="nav-link" href="#Introduction to html"> Introduction to html</a>
+  <a class="nav-link" href="#Basic structure of html">Basic structure of html</a>
+  <a class="nav-link" href="#Commen html elements"> Commen html elements</a>
+  <a class="nav-link" href="#Attribute in html"> Attribute in html</a>
+  <a class="nav-link" href="#Html style"> Html style</a>
+</nav>
+
+<main id="main-doc">
+  <section id="Introduction to html" class="main-section">
```

Comment:
- You have to change the spaces inside your id to underscores `_` as it says in the freecodecamo, and also the text should be the same you have one space in your id at the start.
  - `Introduction to html` should be `Introduction_to_html`
  - Implement the same for all your ids and request for a re-review